libs/ui-toolkit/button/src/lib/button.component.scss

```diff
@@ -21,15 +21,21 @@ button {
 }
 
 button:disabled {
-  background-color: #cacbcc;
+  color: $dark-gray;
+  background-color: $very-light-gray;
   background-image: none;
   cursor: not-allowed;
+  box-shadow: none;
+
+  &:hover {
+    box-shadow: none !important;
```

Comment:
Why the :hover needs the !important while the class doesn't?

Reply:
It had an unnecessary !important applied to the button element when hovering, so I removed it. Thanks!